src/events/enums/events.enum.ts

```diff
@@ -8,5 +8,6 @@ export enum Events {
   CHANNEL_UPDATED = 'channel_updated',
   CHANNEL_DELETED = 'channel_deleted',
   WORKSPACE_UPDATED = 'workspace_updated',
+  USERS_ADDED = 'users_added',
```

Comment:
I believe the name needs to be more explicit about the location where users are being added.